### DISCUSSION TITLE
Continue button not working

### DISCUSSION THREAD

**Discussion by salvobonsma** [community/community#50540] [GitHub Education] [answered]
On the [Student Benefits Application](https://education.github.com/discount_requests/application) page, when clicking the continue button with everything filled out, the button becomes disabled. I also checked for network activity when clicking the button, but there is none.

What should I do from here?

**Comment by auooo** [comment]
I have the same problem, have you solved it?

**Reply by Humbena** [reply]
yes

**Comment by poplpr** [comment]
Me too. If someone has solved this problem, please tell me the solution.

**Reply by Humbena** [reply]
yes

**Comment by YEMIOYE** [comment]
Please fixxxxx

**Comment by OhGaGaGaGa** [comment]
Why this discussion is closed -_-

Again meet this problem ..

**Reply by OhGaGaGaGa** [reply]
Maybe that problem is caused by the location service, which is not very robust in some PC. 
I can visit this application website on my phone and it work well.

**Reply by DemonicDev** [reply]
i have the same issue on phone and both my pcs

**Comment by ghost** [comment]
I have this problem too.

**Reply by OhGaGaGaGa** [reply]
Visit that website on your phone may solve that problem temporarily.

**Reply by DemonicDev** [reply]
it doesnt work on phone for me either @OhGaGaGaGa

**Comment by salvobonsma** [comment, marked as answer]
I used a different browser to solve this problem. I was on Vivaldi, works on Edge.

**Reply by ohadmarbach** [reply]
didn't work still no solution

**Comment by Humbena** [comment]
As of 5/3/2025 I have found the way or my way at least. For me I added a school email straight from my school account (@stu.SanJac.edu). It would not let me click continue. However, I noticed GitHub had a lowercase version of my email saved into the acceptable emails (@stu.sanjac.edu) so I tried adding my same email just with lowercase. It did not work because it recognized the top email as the same thing. So I deleted the top email from my account and re-added it with my lowercase version and boom it worked. I know you’re so happy you found the answer I worked 10 mins to uncover. To all the future data people or whateevr yall wanna be u can thank me with a hi on discord (itztuna)
